xbmc/video/VideoThumbLoader.cpp

```diff
@@ -360,7 +360,15 @@ bool CVideoThumbLoader::LoadItemCached(CFileItem* pItem)
       !setting->FindIntInList(CSettings::VIDEOLIBRARY_THUMB_SHOW_UNWATCHED_EPISODE)
      )
   {
-    pItem->SetArt("thumb", "OverlaySpoiler.png");
+    // use fanart if available
+    if(pItem->HasArt("fanart"))
```

Comment:
Blank between "if" and "(" missing. :-(

Reply:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#34-whitespace

=> "Control statement keywords have to be separated from opening parentheses by one space."

Reply:
I'm sorry that I missed this, I'm used to the autoformatter but my settings are not compatible with other parts of the code guidelines. I made a [pull request](#16019) to fix this and an other instance of the same problem.